src/components/va-back-to-top/test/va-back-to-top.e2e.ts

```diff
@@ -87,82 +40,75 @@ describe('va-back-to-top', () => {
   });
 
   it('reveals when the viewport is below the reveal pixel', async () => {
-    const page = await newE2EPage();
-    await page.setContent(`
-    <main>
-    ${pageContent}
-    <va-back-to-top></va-back-to-top>
-    </main>
-    `);
+    const page = await pageSetup();
     const button = await page.find('va-back-to-top >>> button');
+    const notQuitePastRevealPoint = 199;
```

Comment:
:joy: :100: 

Still, bonus points for using a `revealPoint` variable to declare both `--reveal-breakpoint` in the style and this one here. Just to make sure they're really tied together.